src/Jackett/Indexers/WiHD.cs

```diff
@@ -0,0 +1,702 @@
+            dictionary.Add("565af82d1fd35761568b4590", "565af82d1fd35761568b458f");         // Bluray Remux
+            dictionary.Add("565af82d1fd35761568b4592", "565af82d1fd35761568b4591");         // Bluray 3D
+
+            // Anime
+            dictionary.Add("565af82d1fd35761568b459d", "565af82d1fd35761568b459c");         // 1080P
+            dictionary.Add("565af82d1fd35761568b459f", "565af82d1fd35761568b459e");         // 720P
+            dictionary.Add("565af82d1fd35761568b45a1", "565af82d1fd35761568b45a0");         // HDTV
+            dictionary.Add("565af82d1fd35761568b45a3", "565af82d1fd35761568b45a2");         // Bluray
+            dictionary.Add("565af82d1fd35761568b45a5", "565af82d1fd35761568b45a4");         // Bluray Remux
+            // BUG ~~ Media ID for Anime BR 3D is same as TV BR 3D ~~
+            //dictionary.Add("565af82d1fd35761568b4592", "565af82d1fd35761568b45a6");       // Bluray 3D
+
+            // Other
+            dictionary.Add("565af82d1fd35761568b45b0", "565af82d1fd35761568b45af");         // Apps
+            dictionary.Add("565af82d1fd35761568b45b2", "565af82d1fd35761568b45b1");         // Clips
+            dictionary.Add("565af82d1fd35761568b45b4", "565af82d1fd35761568b45b3");         // Audios Tracks of Movies/TV/Anime
+            dictionary.Add("565af82d1fd35761568b45b6", "565af82d1fd35761568b45b5");         // Documentary
+            dictionary.Add("565af82d1fd35761568b45b8", "565af82d1fd35761568b45b7");         // Bluray (ALL)
+
+            // Check if we know this media ID
+            if (dictionary.ContainsKey(media))
+            {
+                // Due to a bug on tracker side, check for a specific id/name as image is same for TV/Anime BR 3D
+                if(media == "565af82d1fd35761568b4592" && name == "Animations - Bluray 3D")
+                {
+                    // If it's an Anime BR 3D
+                    return "565af82d1fd35761568b45a6";
+                }
+                else
+                {
+                    // Return category ID for media ID
+                    return dictionary[media];
+                }
+            }
+            else
+            {
+                // Media ID unknown
+                throw new Exception("Media ID Unknow !");
+            }
+        }
+
+        /// <summary>
+        /// Output message for logging or developpment (console)
+        /// </summary>
+        /// <param name="message">Message to output</param>
+        /// <param name="level">Level for Logger</param>
+        private void output(string message, string level = "debug")
+        {
+            // Check if we are in dev mode
+            if(DevMode)
+            {
+                // Output message to console
+                Console.WriteLine(message);
+            }
+            else
+            {
+                // Send message to logger with level
+                switch (level)
+                {
+                    default:
```

Comment:
Default will be never used, it's present "just in case of".
I can remove it without problem.
You prefer that i add an option in provider settings to enable "enhanced logging" ?
So by default, this provider will write no logs at all if dev mode is false and enhanced logging false ?